packages/interactor/src/focused.ts

```diff
@@ -1,6 +1,19 @@
+import { perform } from './perform';
+import { ActionFn } from './specification';
+
 /**
  * Helper function for focused filters, returns whether the given element is focused.
  */
 export function focused(element: Element): boolean {
   return element.ownerDocument.activeElement === element;
 }
+
+export const focus: ActionFn<any> =
```

Comment:
This (and the blur) have an `any` type that I am struggling to narrow. Would appreciate any ideas to address that.

Reply:
I think `HTMLElement` is the appropriate type here instead of `any`. That's what you should use, but I'm not sure if there will be any problems due to subtyping/variance on generics, so TypeScript might reject it.

Reply:
`<E extends HTMLElement>` was the ticket here. Thanks!